con-master/src/main/java/ba/codecentric/master/controller/HotelController.java

```
@@ -19,7 +21,7 @@ public HotelController(HotelService hotelService) {
     }
 
     @RequestMapping(value = "/hotel", method = RequestMethod.POST)
-    public Hotel saveHotel(@RequestBody Hotel hotel) {
+    public Hotel saveHotel(@Valid @RequestBody Hotel hotel) throws Exception{
```

Do you need to throw that Exception?